swagger/public_v1/component_schemas/CourseFilter.yml

```
@@ -99,3 +99,7 @@ properties:
     description: "Search radius in miles from given latitude and longitude."
     type: number
     example: 20
+  updated_since:
+    description: "Return courses that only have changed since date (ISO 8601 date format)"
```

```suggestion
    description: "Return courses that only have updated since date (ISO 8601 date format)"
```
Being petty

---

I don't think this reads well.

```suggestion
    description: "Return courses have been updated since the date (ISO 8601 date format)"
```